dmc2gym/wrappers.py

```diff
@@ -131,7 +143,7 @@ def _convert_action(self, action):
         norm_delta = self._norm_action_space.high - self._norm_action_space.low
         action = (action - self._norm_action_space.low) / norm_delta
         action = action * true_delta + self._true_action_space.low
-        action = action.astype(np.float64)
+        action = action.astype(np.float32)
```

Comment:
I'm confused about the dtype conversion here.

Converting to float64 at the beginning kinda makes sense -- I guess you want it high precision as you're about to do some normalization, so as to introduce as little error as possible.

But why is float32 the right thing to convert back to? Can we make it `self._true_action_space.dtype` instead? If that's not the right thing, what is the relevant action space it should belong to?

Reply:
Yeah, _convert_action(action) should output the same dtype as self._true_action_space.dtype.